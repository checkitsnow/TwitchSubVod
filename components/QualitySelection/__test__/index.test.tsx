import { render } from '@testing-library/react';
import QualitySelection from '@/components/QualitySelection';

it('should render Quality Selection correctly', () => {
  const tree = render(<QualitySelection />);
  expect(tree).toMatchSnapshot();
});
