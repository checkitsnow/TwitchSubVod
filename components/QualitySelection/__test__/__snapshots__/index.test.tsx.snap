// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Quality Selection correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styles__Container-ezdgk0-0 dCQtIG"
      >
        <label
          for="quality"
        >
          Select a quality: 
        </label>
        <select
          id="quality"
          name="quality"
        >
          <option>
            Source
          </option>
          <option
            value="720p60"
          >
            720p60
          </option>
          <option
            value="720p30"
          >
            720p30
          </option>
          <option
            value="480p30"
          >
            480p
          </option>
          <option
            value="360p30"
          >
            360p
          </option>
          <option
            value="160p30"
          >
            160p
          </option>
          <option
            value="audio_only"
          >
            audio
          </option>
        </select>
        <div
          class="styles__Container-pgyzn6-0 kCeIyN"
        >
          <div
            class="info-box"
          >
            <p>
              Works best with "Source"
            </p>
          </div>
          <svg
            fill="none"
            height="18"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="18"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="12"
              cy="12"
              r="10"
            />
            <line
              x1="12"
              x2="12"
              y1="16"
              y2="12"
            />
            <line
              x1="12"
              x2="12.01"
              y1="8"
              y2="8"
            />
          </svg>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styles__Container-ezdgk0-0 dCQtIG"
    >
      <label
        for="quality"
      >
        Select a quality: 
      </label>
      <select
        id="quality"
        name="quality"
      >
        <option>
          Source
        </option>
        <option
          value="720p60"
        >
          720p60
        </option>
        <option
          value="720p30"
        >
          720p30
        </option>
        <option
          value="480p30"
        >
          480p
        </option>
        <option
          value="360p30"
        >
          360p
        </option>
        <option
          value="160p30"
        >
          160p
        </option>
        <option
          value="audio_only"
        >
          audio
        </option>
      </select>
      <div
        class="styles__Container-pgyzn6-0 kCeIyN"
      >
        <div
          class="info-box"
        >
          <p>
            Works best with "Source"
          </p>
        </div>
        <svg
          fill="none"
          height="18"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="12"
            cy="12"
            r="10"
          />
          <line
            x1="12"
            x2="12"
            y1="16"
            y2="12"
          />
          <line
            x1="12"
            x2="12.01"
            y1="8"
            y2="8"
          />
        </svg>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
